superset/assets/javascripts/dashboard/reducers/index.js

```diff
@@ -0,0 +1,29 @@
+import { combineReducers } from 'redux';
+import shortid from 'shortid';
+
+import charts from '../../chart/chartReducer';
+import dashboardState from './dashboard';
```

Comment:
I'd update the file to be called `dashboardState` too

Reply:
fixed.